Orange/tests/test_table.py

```diff
@@ -1080,6 +1080,15 @@ def test_calling_new_with_keyword_argument_filename_calls_read_data(
         read_data.assert_called_with(self.filename)
 
 
+class CreateTableWithUrl(TableTests):
+    def load_from_url(self):
+        d1 = data.Table('iris')
+        d2 = data.Table('https://raw.githubusercontent.com/biolab/orange3/'
+                        '04e48a6b11794fd300ad13579e8903e6026f077d/Orange/datasets/iris.tab')
```

Comment:
Wouldn't a link to the latest version of the data set be better than to a version from a specific commit?
i.e. https://raw.githubusercontent.com/biolab/orange3/master/Orange/datasets/iris.tab